lib/swoosh/mailer.ex

```diff
@@ -204,27 +206,33 @@ defmodule Swoosh.Mailer do
     adapter.deliver_many(emails, config)
   end
 
-  @doc """
-  Parse configs in the following order, later ones taking priority:
-
-  1. mix configs
-  2. compiled configs in Mailer module
-  3. dynamic configs passed into the function
-  4. system envs
-  """
+  @doc false
```

Comment:
I imagine these functions are actually private? :)

Reply:
I forgot why I made them public. Probably just so that I could document them.